crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -384,17 +384,34 @@ where
 
     /// Add a leading ZWSP char to this container.
     /// Returns false if no updates was done.
-    /// e.g. first text-like node is a line break
-    /// or a text node that already starts with a ZWSP.
+    /// e.g. first text-like node is already a ZWSP.
     pub fn add_leading_zwsp(&mut self) -> bool {
+        let handle_is_set = self.handle().is_set();
```

Comment:
What situations might the handle not already be set in?

Reply:
https://github.com/matrix-org/matrix-rich-text-editor/pull/412#discussion_r1050751773